2023-03-31 10:03:18,884 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-03-31 10:03:25,553 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-03-31 10:03:33,965 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-03-31 10:03:38,819 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-03-31 10:03:39,037 | ERROR | OMAudit | user=hadoop | ip=10.9.0.17 | op=READ_VOLUME {volume=02888-volume-for-prepare} | ret=FAILURE | VOLUME_NOT_FOUND org.apache.hadoop.ozone.om.exceptions.OMException: Volume 02888-volume-for-prepare is not found
	at org.apache.hadoop.ozone.om.VolumeManagerImpl.getVolumeInfo(VolumeManagerImpl.java:61)
	at org.apache.hadoop.ozone.om.OzoneManager.getVolumeInfo(OzoneManager.java:2582) 
2023-03-31 10:03:43,569 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=GET_SERVICE_LIST {} | ret=SUCCESS |  
2023-03-31 10:03:43,798 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {prevKey=, prefix=, maxKeys=1000, username=hadoop} | ret=SUCCESS |  
2023-03-31 10:03:44,281 | INFO  | OMAudit | user=hadoop | ip=10.9.0.17 | op=LIST_VOLUMES {prevKey=17366-volume-for-cancel, prefix=, maxKeys=1000, username=hadoop} | ret=SUCCESS |  
